docs/api.md

```diff
@@ -477,6 +477,7 @@ An array of all pages inside the browser context.
   - `expires` <[number]> Unix time in seconds.
   - `httpOnly` <[boolean]>
   - `secure` <[boolean]>
+  - `session` <[boolean]>
```

Comment:
How about the opposite? Never return `session`, never set `session`. Rationale: `expires` defines `session` anyways.

Reply:
Fair